Find a functionally equivalent code:android.graphics.Canvas.drawCircle:COMMENT
Method Modifier: public      
Comment:/**
 * Draw the specified circle using the specified paint. If radius is <= 0,
 * then nothing will be drawn. The circle will be filled or framed based
 * on the Style in the paint.
 *
 * @param cx     The x-coordinate of the center of the cirle to be drawn
 * @param cy     The y-coordinate of the center of the cirle to be drawn
 * @param radius The radius of the cirle to be drawn
 * @param paint  The paint used to draw the circle
 */

@@ -1,3 +1,3 @@
 {
-    native_drawCircle(mNativeCanvas, cx, cy, radius, paint.mNativePaint);
+    native_drawCircle(mNativeCanvasWrapper, cx, cy, radius, paint.mNativePaint);
 }

